### DISCUSSION TITLE
an update about the tutorial output of clinicadl tsvtools -h

### DISCUSSION THREAD

**Discussion by mselimata** [aramis-lab/clinicadl#412] [Q&A] [answered]
Hi,
as opposed to the tutorial here [](https://clinicadl.readthedocs.io/en/latest/TSVTools/) clinicadl tsvtools get-labels does not accept caps directory option as input,
version is 1.2.0
here is the output of the help file:
Usage: clinicadl tsvtools get-labels [OPTIONS] BIDS_DIRECTORY

  Get labels in a tsv file.

  This command executes the two following commands:     - `clinica iotools merge-tsv`     - `clinica iotools check-missing-modalities`

  BIDS_DIRECTORY is the path to the BIDS directory.

  Defaults diagnoses are CN and AD.

  Outputs are stored in OUTPUT_TSV.

Options:
  -d, --diagnoses TEXT            Labels selected for the demographic analysis used in the context of Alzheimer's Disease classification.
  -mod, --modality TEXT           Modality to select sessions. Sessions which do not include the modality will be excluded.
  -voi, --variables_of_interest TEXT
                                  Variables of interest that will be kept in the final lists. Will always keep the group (that correspond to the diagnosis in
                                  most case), subgroup (that correspond to the progression of the disease in the case of a progressive disease), age and sex
                                  needed for the split procedure.
  --restriction_tsv TEXT          Path to a TSV file containing the sessions that can be included.
  --keep_smc                      This flag allows to keep SMC participants, else they are removed.
  --merged_tsv TEXT               Path to a TSV file containing the results of clinica iotools merge-tsv command if different of results_directory/merged.tsv
  --missing_mods TEXT             Path to a directory containing the results of clinica iotools missing-modalities command if different of
                                  results_directory/missing_mods/
  --remove_unique_session         This flag allows to remove subjects with a unique session, else they are kept.
  -h, --help

**Comment by camillebrianceau** [comment, marked as answer]
Hi, 

I apologize for the delay in response and appreciate your interest in ClinicaDL.

It is true that this option is no longer supported in the latest versions of ClinicaDL and it has been removed in the last version of the tutorial. This issue will be solved in the next version ([PR](https://github.com/aramis-lab/clinicadl/pull/416)). For now, as stated in the output of the help file, this command requires a BIDS directory to function. However, if you want to restrict your command to specific subjects/sessions from a particular CAPS, you can still use the "--restriction_tsv" option.

The tutorial has just been updated, and as one of the first users, your feedback is really important to us. Please do not hesitate to ask if you have any further questions or feedback.

I hope this helps.

Best regards,
Camille
